portals/devportal/source/src/app/components/Apis/Details/Overview.jsx

```diff
@@ -423,6 +432,34 @@ function Overview() {
                                 )
                             }
                         </Box>
+                        <Box display='flex' flexDirection='row' alignItems='center' mt={2} pr={6}>
+                            {api.advertiseInfo && api.advertiseInfo.advertised
+                                && api.advertiseInfo.originalDevPortalUrl && (
```

Comment:
Remove the label of original devporal url

Reply:
Done from https://github.com/wso2/apim-apps/pull/130/commits/cb08deb420f2cfd43960a6970750298152ce681a